cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/chart.go

```diff
@@ -401,7 +408,54 @@ func availablePackageDetailFromTarball(chartID, url string) (*corev1.AvailablePa
 	}
 	// TODO: (gfichtenholt) LongDescription?
 
-	// note the caller will set pkg.AvailablePackageRef as that information
-	// like namespace and repo name is not included in the tarball
+	// note, the caller will set pkg.AvailablePackageRef namespace as that information
+	// is not included in the tarball
 	return pkg, nil
 }
+
+// packageAppVersionsSummary converts the model chart versions into the required version summary.
```

Comment:
Yeah, it'd be good to avoid the duplication, but don't care if we do that now or later.